src/deviation.jl

```diff
@@ -37,10 +55,22 @@ function sqL2dist{T<:Number}(a::AbstractArray{T}, b::AbstractArray{T})
     return r
 end
 
+
 # L2 distance
+"""
+    L2dist(a, b) -> Float
```

Comment:
My comments also apply to all similar functions below. If you don't feel like copying the equations for all distances, just fix the `Float64` thing.